src/main/java/io/vertx/ext/hawkular/MetricsTypeEnum.java

```diff
@@ -0,0 +1,19 @@
+package io.vertx.ext.hawkular;
+/*
+ * Copyright 2015 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+public enum MetricsTypeEnum {
+    NET_SERVER_TYPE, NET_CLIENT_TYPE, HTTP_SERVER_TYPE, HTTP_CLIENT_TYPE, DATAGRAM_SOCKET_TYPE
```

Comment:
annotated with @vertxgen + add javadoc for the metrics and the various enum entries